runtime/compiler/env/j9method.cpp

```diff
@@ -6491,7 +6492,7 @@ TR_ResolvedJ9Method::getResolvedImproperInterfaceMethod(TR::Compilation * comp,
    if (j9method == NULL)
       return NULL;
    else
-      return createResolvedMethodFromJ9Method(comp, cpIndex, 0, j9method, NULL, NULL);
+      return createResolvedMethodFromJ9Method(comp, cpIndex, (uint32_t)vtableOffset, j9method, NULL, NULL);
```

Comment:
This will still be `0` in the majority of non-(non-final virtual object method) cases (ie: private interface).  Does the `createResolvedMethodFromJ9Method()` already know how to deal with that?

Reply:
`createResolvedMethodFromJ9Method()` doesn't know and it doesn't need to know that. It just creates a `TR_ResolvedJ9Method` object. The parameter `vtableOffset` is stored in `_vTableSlot` of `TR_ResolvedJ9Method` and is only used for non-final virtual method.